actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
+require "action_controller/error"
+
 module ActionController
+  class RedirectFallbackError < Error # :nodoc:
+    def initialize(location = nil)
+      super <<~MSG.squish
+        Unsafe redirect #{location.inspect},
+        use :fallback_location to specify a fallback
```

Comment:
Would not this spam the message in three lines?

Reply:
The heredoc is `squish`ed, so newlines turn into spaces. This makes it more readable in my opinion, but I can jam it into one line if you prefer.

Reply:
Actually, we want to get rid of the custom error so I'll remove this class entirely.